### ISSUE TITLE
CReLU and BatchNorm

### ISSUE THREAD

**Issue by yulizhou** [lxg2015/faceboxes#2]
Hi, I'm reading the paper and curious about your implementation.

CReLU layer seems defined but not used. Instead, the code implements it again in the layer construction. 

Also, the paper has a batch norm layer but it's not implemented.

What is the consideration for this implementation? Better performance?

Thanks

**Comment by lxg2015** [comment]
Whichever way is ok about CReLU .  With the bn layer, faceboxes should have better results, I just forget to add the batch norm layer, thanks

**Comment by abeardear** [comment]
@yulizhou @lxg2015 
Hi, I fix network design in [this repo](https://github.com/xiongzihua/faceboxes), little changes made better performance, such as conv to conv_bn_relu~

**Comment by XiaXuehai** [comment]
@xiongzihua 
hi, I think the predict code is wrong.We can't resize the original image.
See [this repo](https://github.com/XiaXuehai/faceboxes) , the output is closer to the paper.
